README.md

```diff
@@ -16,50 +16,110 @@ When you are done:
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
 # copy & paste your solution here
+def return_max(l):
+  if len(l)==0: 
+    return None
+  else: 
+      l.sort()
+  return l[-1]
 ```
 
 - [Min Value](https://repl.it/@Admin7/minvalue)
 ```python
 # copy & paste your solution here
+def return_min(l):
+  if len(l)==0: 
+    return None
+  else: 
+      l.sort()
+  return l[0]
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
 # copy & paste your solution here
+def return_extended(l, iter):
+  l.extend(iter)
+  return l
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
 # copy & paste your solution here
+def create_matrix(h, w):
+    matrix=[]
+    for i in range(h):
+        row=[]
+        for j in range(w):
+          if w==1:
+            break
```

Comment:
I made a little mistake in test case 2 but now I'm glad I did, because this is a great use of `break`!